src/amber/environment/loader.cr

```diff
@@ -25,11 +21,11 @@ module Amber
     end
 
     private def yml_settings_file
-      @yml_settings ||= "#{@path}/#{@environment}.yml"
+      @yml_settings ||= File.expand_path("#{@path}/#{@environment}.yml")
```

Comment:
I still don't get why we're expanding the path. It results in the same thing anyway.

Reply:
`Amber.environment_path` allows the user to set a different path for the yamls files. For instance `~/Some/Dir`, without expand path this will error.

Reply:
Not sure that should be encouraged... The only way to do that in rails is with a symlink.

Reply:
NVM. I think you're right about this.